roles/telemetry_autoscaling/tasks/creating_stack.yml

```diff
@@ -60,7 +60,7 @@
   until: '"CREATE_COMPLETE" in result.stdout'
     #  failed_when: '"CREATE_COMPLETE" not in result.stdout'
```

Comment:
I think this needs to be updated, but not in this patchset.
IINM, this test does not do what it says.
I need to check the expected output of the resource list, because this might pass when one resource is present.